dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/service/TriBuiltinService.java

```diff
@@ -67,6 +74,11 @@ public void init() {
         }
     }
 
+    public boolean enable(){
+        return config.getBoolean(TRI_BUILTIN_SERVICE_INIT, false);
```

Comment:
Can we check if users has add the `protobuf-java` dependency?

Reply:
First of all, we do not know the real whole sets of versions that works fine, and in the future as protobuf-java update, we may need hava a glance at this; on the other hand, i think give the load permissions to user is a better choice.